soh/soh/Enhancements/randomizer/3drando/fill.cpp

```diff
@@ -1077,7 +1077,7 @@ static void RandomizeDungeonItems() {
 
   //Randomize Any Dungeon and Overworld pools
   AssumedFill(anyDungeonItems, anyDungeonLocations, true);
```

Comment:
Is this still valid or is it missing some of the new fairy/pot locations?

Reply:
Actually I think since I updated the way the dungeon location lists get populated that should be covered already, just a thought that popped into my head when I saw your other changes.

Reply:
yes, reusing your logic which applies for all locations & dungeons already is why this PR gets to mostly remove lines